### ISSUE TITLE
jsdom vs jdon package

### ISSUE THREAD

**Issue by peter-dowling** [mhlabs/cfn-diagram#4]
When I install and run any command including `-v` and `-h` I get an error saying `Error: Cannot find module 'jsdom'`
Looking at the package file, there is reference to a jdom dependency, but not a jsdom one. 
MxGenerator.js looks like it requires jsdom specifically. Is there a mixup somewhere between the two dependencies?
Full stack trace:
```
$ cfn-dia -h
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'jsdom'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/ME/.nvm/versions/node/v10.16.2/lib/node_modules/@mhlabs/cfn-diagram/mxgraph/MxGenerator.js:2:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

**Comment by Zirkonium88** [comment]
Hey @peter-dowling, I just faced the same issue. Simply running `npm install -g jsdom` solved the problem. I'm running the tool on a Mac with wiht OS 10.15.4.

**Comment by ljacobsson** [comment]
Thanks @peter-dowling for reporting and thanks @Zirkonium88 for providing help.

Have just published v1.0.8 that fixes this.
